src/codeu/chat/util/Json.java

```diff
@@ -0,0 +1,58 @@
+package codeu.chat.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+
+/**
+ * This class is used to read in json objects and write json objects to a text file
+ * Created by hv58535 on 6/23/17.
+ */
+public class Json {
+
+    // When running on your own machine change this file path to where you are storing your textfiles.
+    // It is best to save them in the project folder
+    private static final String STORAGE_FILES_PATH = "/Users/hv58535/CodeU-Summer-2017/";
```

Comment:
Instead of storing a full directory path here, use `"./"`, which is the current directory the program is running in - this way if you were to distribute the binary of this software it would work on any computer and no one has to change the source code :)